src/main/java/core/basesyntax/file/writer/FileWriter.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.file.writer;
+
+import core.basesyntax.model.Fruit;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.List;
+
+public interface FileWriter {
+    void write(BufferedWriter bufferedWriter, List<Fruit> fruitList) throws IOException;
```

Comment:
please see my earlier comments, probably you missed some of them
```suggestion
    void write(String path, String data);
```